scalameta/dialects/shared/src/main/scala/scala/meta/Dialect.scala

```diff
@@ -161,7 +161,9 @@ final class Dialect private[meta] (
     // https://docs.scala-lang.org/scala3/reference/other-new-features/control-syntax.html
     val allowQuietSyntax: Boolean,
     // Are binary literals allowed? SIP-42.
-    val allowBinaryLiterals: Boolean
+    val allowBinaryLiterals: Boolean,
+    // Are tracked parameters allowed? docs: https://dotty.epfl.ch/docs/reference/experimental/modularity.html
+    val allowTrackedParameters: Boolean
```

Comment:
I'm sorry, is this feature part of an approved SIP, or released scala version? adding experimental stuff that may or may not be adopted is premature otherwise.

we have not yet implemented 3.4 parsing so I would start with that.

Reply:
I think it would be safe to this to ScalaFuture as it will not be too much problem to remove it in the future. I would agree in terms of bigger features, but here it's rather safe.

I can follow up with 3.4.0 and 3.5.0 dialects later on.